### DISCUSSION TITLE
MEDS/FEMR/CLMBR-T-Base Stack

### DISCUSSION THREAD

**Discussion by kgweber-sag** [som-shahlab/femr#224] [Q&A] [answered]
Hello, all:

When I was working with CLMBR-T-Base this spring, I got all tangled up in versioning incompatibilities between the model itself, FEMR, and MEDS. I see you'vve all been working hard on MEDS (very cool work!) Before I pick this up and start work again, can you please confirm what version of each need to be together for me to test CLMBR-T-Base and tune it over my OMOP dataset?

Your input would be much appreciated

**Comment by EthanSteinberg** [comment, marked as answer]
Last I checked, the versions specified on the huggingface README (`pip install torch==2.1.1 femr==0.2.3 datasets==2.15.0 xformers transformers==4.35.2`) are still correct.

Note that FEMR 0.2.3 requires MEDS v1 (https://github.com/som-shahlab/femr/blob/0.2.3/pyproject.toml#L18).

Note that I would generally recommend training from scratch if at all possible (https://github.com/som-shahlab/femr/blob/0.2.3/tutorials/6_Train%20MOTOR.ipynb). That's the best way to avoid vocabulary issues (where CLMBR-T-Base's vocab doesn't match the vocab of your dataset).

**Reply by kgweber-sag** [reply]
Thanks, Ethan, much appreciated.
